### DISCUSSION TITLE
Jellyseerr installed from Snap but in develop mode?

### DISCUSSION THREAD

**Discussion by LordSith77** [seerr-team/seerr#745] [Q&A] [answered]
I installed Jellyseerr from the Snap store on Ubuntu, but when I launch it and get it all set it says it's in develop mode and out of date. I've tried updating it, to no avail. How can I switch it to not be in developer mode? Thanks.
![image](https://github.com/Fallenbagel/jellyseerr/assets/26206442/ae7b9340-3358-49e8-86d4-799cc1ec38b5)

System: HP-Proliant DL380 G7 with Ubuntu Linux 20.04.6 (Linux 5.15.0-83-generic on x86_64)

**Comment by fallenbagel** [comment, marked as answer]
Snap is no longer supported and a quite a lot commits behind. 

Snap will be supported once we fix our Snap builds.

**Reply by LordSith77** [reply]
Ah, okay thanks. I'll remove that and install it through console then.

**Reply by ChrisMoriarty** [reply]
Good to know, any plans to fix the snap builds?  If not, is there an easy way to export my settings so I can import them into a new install (not snap)?
